### ISSUE TITLE
[BUG] The harvester plugin affects the display of the global settings page

### ISSUE THREAD

**Issue by WuJun2016** [harvester/harvester#3438]
**Describe the bug**

https://github.com/rancher/dashboard/issues/8129

**To Reproduce**

**Expected behavior**

**Support bundle**

**Environment**
 - Harvester ISO version: 
 - Underlying Infrastructure (e.g. Baremetal with Dell PowerEdge R630):

**Additional context**

**Comment by TachunLin** [comment]
Verified **fixed** on `v1.1-ad597fd1-head` (2/10). Close this issue.

### Result
Verified by default Harvester already set resource limit to related Kubevirt's pods 

- `virt-api`
- `virt-controller` 
- `virt-handler`  

with the following resource limit value
  ```
  virt-api: {“cpu”:“400m”,“memory”:“1100Mi”}
  virt-controller: {“cpu”:“800m”,“memory”:“1300Mi”}
  virt-handler: {“cpu”:“700m”,“memory”:“1600Mi”}
  ```


```
$ kubectl get deploy virt-api -n harvester-system  -o yaml

resources:
          limits:
            cpu: 400m
            memory: 1100Mi
          requests:
            cpu: 5m
            memory: 150Mi
```

```
$ kubectl get deploy virt-controller -n harvester-system  -o yaml

resources:
          limits:
            cpu: 800m
            memory: 1300Mi
          requests:
            cpu: 10m
            memory: 150Mi

```

```
$ kubectl get ds virt-handler -n harvester-system  -o yaml

resources:
          limits:
            cpu: 700m
            memory: 1600Mi
          requests:
            cpu: 10m
            memory: 230Mi

```

### Test Information
* Test Environment: 1 node Harvester on local kvm machine 
* Harvester version: `v1.1-ad597fd1-head` (2/10)

### Verify Steps
1. ssh to Harvester management node 
2. Check the kubevirt pods status
  ```
  $ kubectl get pod -n harvester-system | grep virt
  
  harv230213:~ # kubectl get pod -n harvester-system | grep virt
  virt-api-7cfd78b999-sk2s6                               1/1     Running   0          137m
  virt-controller-5d54b8b9bf-szh86                        1/1     Running   0          137m
  virt-controller-5d54b8b9bf-zz72s                        1/1     Running   0          137m
  virt-handler-6r8s8                                      1/1     Running   0          137m
  virt-operator-7bdb8d7bff-dtl7t                          1/1     Running   0          139m
  ```
3. Check the virt-api yaml content 
  ```
  kubectl get deploy virt-api -n harvester-system  -o yaml
  ```
4. Check the virt-controller yaml content  
  ```
  kubectl get deploy virt-controller -n harvester-system  -o yaml
  ```
5. Check the virt-handler yaml content 
  ```
  kubectl get ds virt-controller -n harvester-system  -o yaml
  ```

**Comment by TachunLin** [comment]
Verified **fixed** on Rancher `v2.7.1` and `v2.6.10` with Harvester master-6fd9a5bb-head (2/10). 
Close this issue.

### Result
Verified on Rancher v2.7.1, we update the ui-dashboard-index to the latest change. 
And set with Harvester master using the previous ui-dashboard-index value 

* This global setting can display correctly after navigate from virtualization management 

https://user-images.githubusercontent.com/29251855/218666683-ea44631f-d7c2-4e46-aa09-2989450a246b.mp4



* Compared to Rancher v2.7.1 with default  ui-dashboard-index, we would possible encounter the crash page issue


https://user-images.githubusercontent.com/29251855/218667864-33379cd0-dd15-452a-8a9f-628b1afff76f.mp4




### Test Information
* Test Environment: 1 node Harvester on local kvm machine
* Rancher version: `v2.7.1` and `v2.6.10` 
* Harvester version: master-6fd9a5bb-head (2/10) 
* Harvester ui-dashboard-index: https://releases.rancher.com/harvester-ui/dashboard/release-harvester-v1.1/index.html 
* Rancher ui-dashboard-index: https://releases.rancher.com/harvester-ui/dashboard/latest/index.html 
* Rancher ui-offlince-preffered: Remote

### Verify Steps
1. Update Harvester `ui-dashboard-index` value to https://releases.rancher.com/harvester-ui/dashboard/release-harvester-v1.1/index.html 
2. Update Rancher `ui-dashboard-index` value to https://releases.rancher.com/harvester-ui/dashboard/release-harvester-v1.1/index.html 
3. Import Harvester to Rancher
4. Access Harvester in virtualization management 
5. Navigate images, volumes and virtual machine page 
6. Navigate to Rancher global settings from hamburger menu
